### PR TITLE
Move the Globalization and Encoding contracts to System.Runtime

### PR DESCRIPTION
**Pull request by tarekgh**
This change is to add the globalization and encoding to System.Runtime and update
System.Globalization.* and System.Text.Encoding contarcts too just type forward instead of
having any implementation

### CODE REVIEW

**Comment by tarekgh** [comment]
@weshaggard @ericstj could you please have a look? 

CC @danmosemsft

File: src/System.Runtime/ref/System.Runtime.cs [hunk 1]
```diff
@@ -2995,33 +3342,46 @@ public partial class CompareInfo
         public virtual int Compare(string string1, string string2) { return default(int); }
         public virtual int Compare(string string1, string string2, System.Globalization.CompareOptions options) { return default(int); }
         public override bool Equals(object value) { return default(bool); }
+        public static System.Globalization.CompareInfo GetCompareInfo(int culture) { return default(System.Globalization.CompareInfo); }
         public static System.Globalization.CompareInfo GetCompareInfo(string name) { return default(System.Globalization.CompareInfo); }
+        // Temproray disable the API's depends on System.Reflection.Assembly till we get it defined in System.Runtime
```
Comment (tarekgh) [inline src/System.Runtime/ref/System.Runtime.cs]: >System.Reflection.Ass [](start = 50, length = 21)

These are commented till we get System.Reflection.Assembly moved to System.Runtime

Comment (danmosemsft) [inline src/System.Runtime/ref/System.Runtime.cs]: please make sure there's an issue assigned to someone for that.

---
In reply to: [76645138](https://github.com/dotnet/corefx/pull/11226#discussion_r76645138) [](ancestors = 76645138)

Comment (tarekgh) [inline src/System.Runtime/ref/System.Runtime.cs]: I believe @sepidehMS already has a PR to move System.Reflection.Assembly to System.Runtime. so I think we'll have the change soon

File: src/System.Globalization.Calendars/src/project.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "frameworks": {
+    "netstandard1.7": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.CoreCLR": "1.1.0-beta-24426-03"
+      }
+    },
     "netstandard1.3": {
```
Comment (weshaggard) [inline src/System.Globalization.Calendars/src/project.json]: You can remove the netstandard1.3 and netcore50 sections in this file.

File: src/System.Runtime/ref/System.Runtime.cs (hunk 1 above)
Comment (weshaggard) [inline src/System.Runtime/ref/System.Runtime.cs]: I think @danmosemsft is just meaning we need to have an issue tracking the exposing of these commented out members once both glob and reflection is merged.

File: src/System.Text.Encoding/src/project.json [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
   "frameworks": {
+    "netstandard1.7": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.CoreCLR": "1.1.0-beta-24426-03"
+      },
+      "imports": [
+        "dotnet5.4"
+      ]
+    },
     "netstandard1.3": {
```
Comment (weshaggard) [inline src/System.Text.Encoding/src/project.json]: Clean-up removed TFMs.

**Comment by weshaggard** [comment]
Other then a couple clean-up comments LGTM once CI is green.

**Comment by tarekgh** [comment]
test Innerloop Ubuntu14.04 Debug Build and Test please

**Comment by tarekgh** [comment]
test  Innerloop Ubuntu14.04 Debug Build and Test please

**Comment by tarekgh** [comment]
test  Innerloop Ubuntu14.04 Release Build and Test please

**Comment by tarekgh** [comment]
test  Innerloop CentOS7.1 Debug Build and Test please
test Innerloop CentOS7.1 Release Build and Test please

**Comment by tarekgh** [comment]
test  Innerloop OSX Release Build and Test please

**Comment by tarekgh** [comment]
test  Innerloop OSX Debug Build and Test please

**Comment by tarekgh** [comment]
test Innerloop CentOS7.1 Debug Build and Test please
test  Innerloop CentOS7.1 Release Build and Test please

File: src/System.Runtime/ref/System.Runtime.cs (hunk 1 above)
Comment (tarekgh) [inline src/System.Runtime/ref/System.Runtime.cs]: I created the issue https://github.com/dotnet/corefx/issues/11241 to track it.

**Pull request merged**